src/dune_rules/install_rules.ml

```diff
@@ -376,7 +393,12 @@ end = struct
               each section. It feels like we should just do this here once and
               for all. *)
            List.sort entries ~compare:(fun (_, a) (_, b) ->
-               Install.Entry.compare Path.Build.compare a b))
+               let to_entry = function
```

Comment:
This pattern is repeated several times, I suggest creating a `Install.Metadata.entry` function